.dockerignore

```diff
@@ -1,8 +0,0 @@
-.git
-**/.DS_Store
-node_modules
-bower_components
-app/libs
-app_build
-lib
-deploy.sh
```

Comment:
while this file was delete.?
it's rquired to avoid certains file like the `.git` folder